modules/social_features/social_event/config/install/field.storage.node.field_enroll_method.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 8bc5637a-1e6a-4cf7-a1ed-aa0db3fc6202
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - options
+id: node.field_enroll_method
+field_name: field_enroll_method
+entity_type: node
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: '1'
```

Comment:
I think we should step away from using `strings` here, but rather use `integers`. Imho it's cleaner to do this with integers, also as these are simply numbers?
This would mean that the checks we added should also be changed to check on integers instead of the string value.

I know that for the `field_enrollment_status` we also use `strings`, but I think we should change that at a different moment.

Reply:
We should also make this constants with a readable format.

Example:
```
class GroupInvitation extends GroupContentEnablerBase {
  /**
   * Invitation created and waiting for user's response.
   */
  const INVITATION_PENDING = 0;
  /**
   * Invitation accepted by user.
   */
  const INVITATION_ACCEPTED = 1;
  /**
   * Invitation rejected by user.
   */
  const INVITATION_REJECTED = 2;
```